app/src/main/java/nu/yona/app/YonaApplication.java

```diff
@@ -202,5 +209,53 @@ public static Tracker getTracker()
 		}
 		return mContext.tracker;
 	}
+
+	protected OpenVPNService mService;
+
+	private final ServiceConnection mConnection = new ServiceConnection()
```

Comment:
Same here. Next, initialize it through a call to a private method rather than putting all of this inline here. Also strip the ``m`` prefix, as that's not a convention we apply.